### ISSUE TITLE
Use case doc should not propose a solution

### ISSUE THREAD

**Issue by marcoscaceres** [WICG/cq-usecases#22]
I'm a bit bothered to see a solution proposal in the use cases document. The use cases document should really only make recommendations about what the requirements are, but not propose a solution itself. What is could do is instead point to proposed solutions - and discuss the pros and cons of those.

**Comment by Wilto** [comment]
Yeah; I was talking about this with @zcorpan the other day. It’s in the works.

**Comment by zcorpan** [comment]
http://ircbot.responsiveimages.org/bot/log/respimg/2014-11-11#T100310

**Comment by marcoscaceres** [comment]
I can try to be of assistance if needed.

**Comment by Wilto** [comment]
Absolutely, @marcoscaceres. You know we always need all the help we can get.

**Comment by eeeps** [comment]
@marcoscaceres ircbot.responsiveimages is down (dead?). Reading the doc, I'm not clear on which part or parts cross this line. Something here, maybe?

> This document presents some of the use cases that “element queries” would solve, in allowing authors to define styles (chiefly, layouts) within an individual module on the basis of the size of the module itself rather than the viewport.

**Comment by marcoscaceres** [comment]
@eeeps, unfortunately, I don't manage the IRC bot :( I can't remember who does. 

In any case, the OP probably stands. @yoavweiss listed a bunch of question over in #49 that the document needs to answer instead.
